File: lookups/v1/phone_number/fetch-lookup-addons-nomorobo/fetch-lookup-addons-nomorobo.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Lookups.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var addOns = new List<string> {
            "nomorobo_spamscore"
        };

        var phoneNumber = PhoneNumberResource.Fetch(
            addOns: addOns,
            pathPhoneNumber: new Twilio.Types.PhoneNumber("+19892008374")
        );

        Console.WriteLine(phoneNumber.CallerName);
    }
}

File: lookups/v1/phone_number/fetch-lookup-addons-nomorobo/fetch-lookup-addons-nomorobo.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

phone_number = @client.lookups
                      .phone_numbers('+19892008374')
                      .fetch(add_ons: 'nomorobo_spamscore')

puts phone_number.caller_name
